pages/complete-login.js

```diff
@@ -45,7 +47,10 @@ const CompleteLoginPage = () => {
         setIsLoading(false)
       }, 1000)
       setTimeout(() => {
-        router.push('/dashboard')
+        // Set cached dest no matter what and redirect to one if it existed
+        setFlossbankDest('')
+        if (flossbankDest) router.push(flossbankDest)
```

Comment:
Guessing this works bc of nextTick internals but the code is very misleading here-- setting it blank and then pushing to it. Does it not work the "expected" way (flipped)?